### PR TITLE
[Task] Add po editor sync

### LINKED ISSUE #2: [Github Action] Setup PO editor sync
https://github.com/pimcore/poeditor-export-action

### PR DESCRIPTION
**Pull request by mattamon**
## Changes in this pull request  
Resolves #2 

## Additional info  
Setting up po editor sync

### CODE REVIEW

File: .github/workflows/poeditor-export.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: "Trigger POEditor Translations Export"
+
+on:
+    workflow_dispatch:
+    push:
+        branches:
+            - "[0-9]+.x"
+        paths:
+            - 'translations/admin.en.yaml'
```
Comment (martineiber) [inline .github/workflows/poeditor-export.yaml:9]: Should we add `admin_ext.en.yaml` as well?
```suggestion
            - 'translations/admin.en.yaml'
            - 'translations/admin_ext.en.yaml'
```

**Review by martineiber — CHANGES_REQUESTED** [review]

File: .github/workflows/poeditor-export.yaml (hunk 1 above)
Comment (mattamon) [reply .github/workflows/poeditor-export.yaml:9]: Isn't this covered by the "extended" part in the csv?
https://github.com/pimcore/poeditor-export-action/pull/14/files#diff-3e8dca5aec3d81be5ce92295a045dea8f99d41b34caa0bc278e750e56995fbc5R21

But I am not sure either

Comment (mattamon) [reply .github/workflows/poeditor-export.yaml:9]: Still not sure how this is handled

**Review by martineiber — APPROVED** [review]
LGTM!

**Pull request merged**